libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/MapboxCarNavigationManager.kt

```diff
@@ -42,19 +43,26 @@ class MapboxCarNavigationManager internal constructor(
         onUpdateTrip(trip)
     }
 
-    private val tripSessionStateObserver = TripSessionStateObserver { tripSessionState ->
-        logAndroidAuto("$LOG_CATEGORY tripSessionStateObserver state: $tripSessionState")
-        when (tripSessionState) {
-            TripSessionState.STARTED -> navigationManager.navigationStarted()
-            TripSessionState.STOPPED -> navigationManager.navigationEnded()
+    private var inActiveNavigation = false
```

Comment:
Agree on keeping it. Also noting that other parts of AA will crash if we're not keeping track of it

https://github.com/mapbox/mapbox-navigation-android/pull/7366/files#diff-85aa933b9a6ac3e143013c4cb6fb6dce239a6da54867c00170385564336f8547R111-R112
```
            // There is no way to know if NavigationManager isNavigating and it will crash if false
            // https://issuetracker.google.com/u/0/issues/260968395
```